pytest_fixtures/broker.py

```diff
@@ -39,3 +41,17 @@ def rhel77_contenthost_class(request):
     with VMBroker(host_classes={'host': ContentHost}, **BROKER_RHEL77) as host:
         request.cls.content_host = host
         yield
+
+
+@pytest.fixture
+def satellite_latest():
+    """A fixture that provides a latest Satellite"""
+    with VMBroker(host_classes={'host': Satellite}, workflow='deploy-sat-lite') as sat:
```

Comment:
These workflow names should be pulled from configuration instead of hard coded. Not a blocker for this initial design, but moving forward we need to keep the broker provider dependencies decoupled from the testing framework.

Reply:
that's a good idea, but let's wait until after dynaconf phase 2, as i'd like to nest some of those.